opennlp-docs/src/docbkx/evaltest.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
+"http://www.oasis-open.org/docbook/xml/4.4/docbookx.dtd"[
+]>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<chapter id="opennlp.evaltest">
+<title>Evaluation Test Data</title>
+	<section id="opennlp.evaltest.whatisit">
+		<title>What is it ?</title>
+		<para>
+			Eval test data is the data that helps with the evaluation tests to evaluate the functionality and
+			performance of OpenNLP.
+			These tests ensure reliability and can help identify potential bugs, errors, or performance issues.
+		</para>
+		<para>
+			The evaluation tests leverage the k-fold cross-validation procedure.
+			This technique works by dividing the eval-data into 'k' equally sized parts or folds.
+			The algorithm is then trained on 'k-1' of the folds and tested on the remaining fold.
+			This process is repeated 'k' times, so that each of the k-folds is used exactly once as the test data,
+			and the results of each fold are combined to produce an overall estimate of the algorithm's performance.
+		</para>
+	</section>
+	<section id="opennlp.evaltest.whereisit">
+		<title>Where is it?</title>
+		<para>
+			OpenNLP Eval Test Data is available at <ulink url="https://nightlies.apache.org/opennlp/opennlp-data.zip">
```

Comment:
Maybe link to the folder? <https://nightlies.apache.org/opennlp/

So there are no accidental clicks to a 3.2 GB file :sweat_smile: and also if we change the name of the file, or add anything else.